pages/style-guide/prose.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Prose Style Guide
+---
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/), and the beginning of each article contains some [YAML](http://www.yaml.org/) "front matter" that contains metadata used when the article is published.
+
```

Comment:
@scottgonzalez Do you agree that we can erase from the first comma (after link to explanation of Github Flavored Markdown) to the end of the paragraph? 

The paragraph would look like this:

Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/)